ui/shared/components/panel/variants/FamilyVariantTags.jsx

```diff
@@ -182,9 +182,11 @@ VariantTagField.propTypes = {
   family: PropTypes.object.isRequired,
 }
 
-const VariantNoteField = ({ action, note, variant, family, ...props }) => {
+const VariantNoteField = ({ action, note, variant, family, showInLine, noModal, ...props }) => {
```

Comment:
you aren't using `showInLine` and `noModal` so theres no reason to declare them here